5.5/Dockerfile.rhel7

```diff
@@ -0,0 +1,20 @@
+FROM rhel7
+
+ADD mysql /opt/openshift/
+ENV PATH /opt/openshift/bin:$PATH
+
+WORKDIR /opt/openshift
+RUN create_user.sh
+
+RUN yum install -y yum-utils hostname && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --enable rhel-7-server-optional-rpms && \
+    yum install -y --setopt=tsflags=nodocs mysql55 && \
+    yum clean all
+
+VOLUME ["/var/lib/mysql"]
+
+ENTRYPOINT ["entrypoint.sh"]
+
+EXPOSE 3306
```

Comment:
Why? After some thought, I think it's best right after entrypoint. why have intermediate images with exposed port when there is no chance of it being used?